### DISCUSSION TITLE
pydantic 1.9.0 does not support python 3.6

### DISCUSSION THREAD

**Discussion by fraser-langton** [fastapi/fastapi#6339] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
requires = [
    "starlette ==0.17.1",
    "pydantic >=1.6.2,!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<2.0.0",
]
```

### Description

[pydantic 1.9.0 dropped support for Python 3.6](https://github.com/samuelcolvin/pydantic/commit/6f46a5a1460728868f860840e3f6186aa40403d8) so for FastAPI to continue to support 3.6 the toml must be `"pydantic >=1.6.2,!=1.7,!=1.7.1,!=1.7.2,!=1.7.3,!=1.8,!=1.8.1,<1.9.0"`?

### Operating System

Windows

### FastAPI Version

0.72.0

### Python Version

3.6.8

### Additional Context

issue arose from a serverless build error using `serverless-python-requirements`:
```
 {
    cause: [Error: ENOENT: no such file or directory, open '.serverless/requirements/pydantic/types.py'] {
      errno: -2,
      code: 'ENOENT',
      syscall: 'open',
      path: '.serverless/requirements/pydantic/types.py'
    },
    isOperational: true,
    errno: -2,
    code: 'ENOENT',
    syscall: 'open',
    path: '.serverless/requirements/pydantic/types.py'
  }
```

**Comment by Kludex** [comment, marked as answer]
Pydantic 1.9.0 still supports Python 3.6. 

See [this](https://pypi.org/project/pydantic/#files) and image below.

<img width="1278" alt="image" src="https://user-images.githubusercontent.com/7353520/157436153-646c5129-678b-4ee9-ba5a-b82b9dc39e42.png">

Next Pydantic version will not support Python 3.6.
